dali/operators/random/normal_distribution_op_test.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <random>
+#include <algorithm>
+#include "dali/operators/random/normal_distribution_op.cuh"
+
+namespace dali {
+
+class TestBlockDistribution : public ::testing::Test{
+ protected:
+  std::vector<int> distribute(const std::vector<int> &sizes) {
+    TensorListShape<1> shape(sizes.size());
+    for (int s = 0; s < shape.size(); ++s) shape.set_tensor_shape(s, {sizes[s]});
+    return detail::DistributeBlocksPerSample(shape, block_size_, max_blocks_).first;
+  }
+
+  const int block_size_ = 256;
+  const int max_blocks_ = 1024;
+};
+
+TEST_F(TestBlockDistribution, Uniform) {
```

Comment:
A test case for when the numbers of blocks doesn't exceeded max_blocks would be nice to have.

Reply:
added